### PR TITLE
Array rotation

### LINKED ISSUE #1051: Array Rotation in ruby
i would like to add array rotation in ruby.

### PR DESCRIPTION
**Pull request by AnshitaVishwa**
Fixes #1051 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Sample Input / Output.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

#### Changes proposed in this pull request:

- Implemented Array Rotation in Ruby

#### Languages Used:
- Ruby

#### Files Added:
- Array_Rotation.rb

### CODE REVIEW

**Comment by AnshitaVishwa** [comment]
Kindly review it.

File: Array_Rotation/Array_Rotation.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    temp = arr[0]
+    for i in 0..n-2
+        arr[i] = arr[i+1]
+    end
+    arr[n-1] = temp
+    return arr
+end
+
+# Function for the Number of times Rotation to be performed
+def Rotation(input , d , n)
```
Comment (jainaman224) [inline Array_Rotation/Array_Rotation.rb]: Please use better variable name than `d`.

File: Array_Rotation/Array_Rotation.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    temp_arr = input
+    for i in 0..d-1
+        temp_arr = Left_Array_Rotation(temp_arr , n)
+    end
+    for i in 0..n-1
+        print(temp_arr[i].to_s + " ")
+    end
+end
+
+input = [1,3,8,5,9]
```
Comment (MastersAbh) [inline Array_Rotation/Array_Rotation.rb]: give spaces after comma

File: Array_Rotation/Array_Rotation.rb [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# Function for the Array Rotation
```
Comment (MastersAbh) [inline Array_Rotation/Array_Rotation.rb]: Add comments explaining the problem statement

File: Array_Rotation/Array_Rotation.rb [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+def Left_Array_Rotation(arr , n)
+    temp = arr[0]
+    for i in 0..n-2
+        arr[i] = arr[i+1]
+    end
+    arr[n-1] = temp
+    return arr
+end
```
Comment (KanchanThareja) [inline Array_Rotation/Array_Rotation.rb]: Yes, please explain through comments what is left_array_rotation doing.
Purpose of using the function and so on.

**Comment by KanchanThareja** [comment]
@AnshitaVishwa Once done squash your commits.

**Comment by KanchanThareja** [comment]
@jainaman224 I approve this PR. Please take a final look.

**Pull request merged**